Formula/global.rb

```diff
@@ -86,7 +89,7 @@ def pyfunc ():
            pyvar = py2func()
     EOS
 
-    assert shell_output("#{bin}/gtags --gtagsconf=#{share}/gtags/gtags.conf --gtagslabel=pygments .")
+    assert shell_output("#{bin}/gtags --gtagsconf=#{share}/gtags/gtags.conf --gtagslabel=pygments")
```

Comment:
```suggestion
    system bin/"gtags", "--gtagsconf=#{share}/gtags/gtags.conf", "--gtagslabel=pygments"
```
This is a bit better if we just want to check that the command executed successfully (i.e. exit code of `0`). Same comment for the similar lines below.